### ISSUE TITLE
Can not open VT's from right Ctrl>Alt>F3-6

### ISSUE THREAD

**Issue by benbullard79** [OpenMandrivaAssociation/distribution#2806]
**OpenMandriva version**: Cooker

**Describe the bug**: Can not open VT's from Plasma desktop using right Ctrl>Alt>F3-6. 

**Steps to reproduce**: In fully dsync'd Cooker plasma desktop with either plasma-x11 or plasma-wayland try to open a VT from right side of kbd Ctrl>Alt>F3-6. 
**Observed behavior**: VT's do not open and it is very obvious that the different F* keys now do other tasks.

**Expected behavior**: I have been opening VT's basically this way since Mandriva, I know some folks do this with left side.

**Additional comment**: They do open still form left side of kbd. Since to me this is a very noticeable change in behavior I am filing this report. I do wonder if this might be a change upstream. Looked briefly and did not find anything yet.

If it is a change and I missed that I believe we need to put something in Release Notes for next Rolling and next Rock versions.

**Logs and screenshots if relevant**  Not sure how to, or if more documentation is really needed. I know other people on Cooker IRC see this.

Omdv bug report log, this does contain journal log and probably x-session log.

[omdv-bug-report-20220630154342.log](https://github.com/OpenMandrivaAssociation/distribution/files/9023335/omdv-bug-report-20220630154342.log)

**Comment by tpgxyz** [comment]
Works here. Are you sure you have you cooker fully synced ? When you press CTRL+ALT+F a dedicated systemd service is spawned /usr/lib/systemd/system/getty@.service Please make sure your agetty is in right place and /usr/sbin is a symlink to /usr/bin

What does says in your case:
```
[tpg@omv-rockpro64 ~]# which agetty                                                                                                                                                                              
/usr/bin/agetty
```

**Comment by benbullard79** [comment]
```
$ which agetty
/usr/bin/agetty

```
The issue is not that there are 2 sets of Ctrl>Alt keys, left side and right side. If I try on desktop with nothing open the right side Ctrl>Alt>F3-F6 keys do nothing. The left side Ctrl>Alt>F3-F6 keys open the appropriate VT as one would expect.

If I have either Falkon or Firefox open right side Ctrl>Alt>F5 refreshes the open browser page. Right side Ctrl>Alt>F6 highlights the address in browser location bar. I see this part only with those 2 browsers, not with Google Chrome, Dolphin, or Gwenview.

If someone reading this report finds this weird, so do I. At first notice I thought this was something broken but it now seems like some change in behavior. I have not found anything about this on internet yet.

**Comment by benbullard79** [comment]
I checked a different kbd in my desktop. I checked  in 4.3, Rolling, and Cooker partitions on desktop. I checked in Rolling and Cooker partitions on ancient notebook. Behavior is same in all cases.

_So now it looks another case of me being wrong about something._

Now what bugs me is I do not recall seeing in any documentation anywhere anything about some difference in behavior if one uses the keys on the left or the right.
